### PR TITLE
Add MangaFast

### PR DESCRIPTION
**Pull request by AbdullahM0hamed**

### CODE REVIEW

File: src/en/mangafast/src/eu/kanade/tachiyomi/extension/mangafast/MangaFast.kt [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+    override fun popularMangaSelector() = "li.ranking1"
+
+    override fun popularMangaFromElement(element: Element) = SManga.create().apply {
+        setUrlWithoutDomain(element.select("a").attr("href"))
+        title = element.select("h4").text().trim()
+        thumbnail_url = element.select("img").attr("abs:data-src").substringBeforeLast("resize")
+    }
+
+    override fun popularMangaNextPageSelector() = "Unused"
```
Comment (SnakeDoc83) [inline src/en/mangafast/src/eu/kanade/tachiyomi/extension/mangafast/MangaFast.kt:33]: Do `popularMangaNextPageSelector(): String? = null`

File: src/en/mangafast/build.gradle [hunk 2]
```diff
@@ -0,0 +1,12 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+
+ext {
+    appName = 'Tachiyomi: MangaFast'
```
Comment (SnakeDoc83) [inline src/en/mangafast/build.gradle:5]: Gradles were changed, it's `extName = 'Extension Name'` now.

Comment (AbdullahM0hamed) [reply src/en/mangafast/build.gradle:5]: I see I am living in the past.

**Comment by SnakeDoc83** [comment]
You missed an "en" folder in the folder structure, there's also a nullability warning for the date parsing.

**Pull request merged**